style.css

```diff
@@ -45,11 +45,59 @@ header {
   color: #6070ff;
 }
 
+.mobile-menu-m ul li {
+  margin-bottom: 40px;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 20px;
+}
+
+.mobile-menu-m ul {
+  padding: 70px 15px 24px 45px;
+  width: 100%;
+  list-style-type: none;
+  color: white;
+}
+
+.mobile-menu-m {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  z-index: 3;
+  display: flex;
+  text-align: left;
+  transform: translateX(-100%);
+  transition: transform 0.6s cubic-bezier(0.19, 1, 0.22, 1);
+  backdrop-filter: blur(5px);
+  background: #6070ff;
+  mix-blend-mode: multiply;
+}
```

Comment:
- [ ] Great job adding this cool menu bar but it seems like some of the text is overlapping with the content in the back so I highly advise you to fix it and make it more like the Figma design:
   - Your design:
     ![image](https://user-images.githubusercontent.com/104968236/232832490-16b94dbb-6dbf-4036-a8da-c98cd29c66f4.png)
    - You can see the "Contact" text is overlapping with the text from the background so please make sure to fix it. ✔
    - Figma design:
    ![image](https://user-images.githubusercontent.com/104968236/232833310-0a3ef00c-b2b8-49e5-b324-1a35379b97d2.png)
    - As you can see from the above design there seems to be a blur on the background so they don't overlap. It's okay if you don't implement the blur exactly but at least make sure the text on the menu bar is not overlapping.